codecov/views.py

```diff
@@ -28,3 +30,31 @@ def redirect_app(request):
     """
     app_domain = settings.CODECOV_DASHBOARD_URL
     return HttpResponseRedirect(app_domain + request.path.replace("/redirect_app", ""))
+
+
+class RepositoryAutoCompleteSearch(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        # must be authenticated to query
+        if not self.request.user.is_authenticated:
```

Comment:
i think if you do `self.request.user.is_staff` instead of `is_authenticated` it's just users who can access the admin interface